### PR TITLE
[Minor] Fix text and make text fit in one line an a typical laptop screen

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

English was not quite correct and one word over a line of text.

### Why are the changes needed?

Looks nicer.

Fix: # (issue)

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Tested locally

### CODE REVIEW

**Comment by justinmclean** [comment]
image

File: web/app/(home)/MetalakesView.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ const MetalakesView = () => {
       <Grid item xs={12}>
         <Typography className={'twc-mb-4 twc-text-[1.375rem] twc-font-bold'}>Metalakes</Typography>
         <Typography sx={{ color: 'text.secondary' }}>
-          A metalake is the top-level container for data in Gravitino. With a metalake, Gravitino provides a 3-level
-          namespace for organizing data: catalog, schemas (also called databases), and tables / views.
+          A metalake is the top-level container for data in Gravitino. Within a metalake, Gravitino provides a 3-level
+          namespace for organizing data: catalog, schemas /databases, and tables / views.
```
Comment (jerryshao) [inline web/app/(home)/MetalakesView.js:19]: Can you please add whitespace before "databases", like `/ databases`?

Comment (justinmclean) [reply web/app/(home)/MetalakesView.js]: correct usage to to have no spaces

Comment (justinmclean) [reply web/app/(home)/MetalakesView.js]: https://www.grammarly.com/blog/slash/

**Review by jerryshao — APPROVED** [review]

**Pull request merged**